### PR TITLE
Add new collections to projects

### PR DESCRIPTION
**Pull request by jillr**
Depends-On: #573

### CODE REVIEW

**Comment by jillr** [comment]
@alinabuzachis @Akasurde please review - I will rebase after 573 merges

**Review by Akasurde — APPROVED** [review]

**Review by alinabuzachis — APPROVED** [review]
Same remark for eda, but other than that LGTM!

File: zuul.d/projects.yaml [hunk 1]
```diff
@@ -471,3 +471,41 @@
...
+
+- project:
+    name: ansible-collections/ansible_hub
+    default-branch: main
+    templates:
+      - publish-to-galaxy
+      - publish-to-automation-hub
+
+- project:
+    name: ansible-collections/ansible/event-driven-ansible
```
Comment (abikouo) [inline zuul.d/projects.yaml:483]:
```suggestion
    name: ansible/event-driven-ansible
```

**Review by abikouo — CHANGES_REQUESTED** [review]

File: zuul.d/projects.yaml [hunk 2]
```diff
@@ -471,3 +471,41 @@
...
+    default-branch: main
+    templates:
+      - publish-to-galaxy
+      - publish-to-automation-hub
```
Comment (alinabuzachis) [inline zuul.d/projects.yaml:487]: We probably only need `publish-to-automation-hub`.  cc @Alex-Izquierdo

Comment (Alex-Izquierdo) [reply zuul.d/projects.yaml:487]: That's correct. Our GH workflow already publishes to galaxy.

**Review by alinabuzachis — APPROVED** [review]

**Pull request merged**